IGC/WrapperLLVM/include/llvmWrapper/IR/IRBuilder.h

```diff
@@ -301,6 +301,14 @@ namespace IGCLLVM
 
 #endif
 
+#if LLVM_VERSION_MAJOR >= 14
+        Value* CreatePtrDiff(Value *LHS, Value *RHS, const Twine &Name = "") {
+          auto *PtrTy = cast<PointerType>(LHS->getType());
+          Type *Ty = PtrTy->getElementType();
+          return llvm::IRBuilder<T, InserterTyDef()>::CreatePtrDiff(Ty, LHS, RHS, Name);
+        }
+#endif
```

Comment:
Does `CreatePtrDiff` exist in LLVM < 14? Will this be available on LLVM < 14?

Reply:
> Does `CreatePtrDiff` exist in LLVM < 14? Will this be available on LLVM < 14?

Upon investigation, it seems like this API exists since LLVM 3.x era. So should I just remove the `#ifdef` altogether?

Reply:
Sure, thanks!

Reply:
I realized I have to actually add this `#ifdef` back since the reason I added this in the first place was because the API changed in LLVM 14 not because the API was not available.